lib/libpreopen.c

```diff
@@ -187,6 +189,97 @@ po_find(struct po_map* map, const char *path, cap_rights_t *rights)
 	return match;
 }
 
+void 
+po_errormessage(const char *msg){
+	perror(msg);
+  	exit(EXIT_FAILURE);
```

Comment:
Library code shouldn't exit: it should return an error value so that the caller can decide what to do.